src/Avatar/Accessory.tsx

```diff
@@ -14,10 +14,15 @@ import { RneFunctionComponent } from '../helpers';
 
 export type AccessoryProps = Partial<IconProps> &
   Partial<ImageProps> & {
+    /**Add underlay color to the accessory of avatar. */
```

Comment:
```suggestion
    /** Add underlay color to the accessory of avatar. */
```

Reply:
@khushal87 Can you make this change everywhere? You can find and replace all.

Reply:
@dhruvdutt Done, thanks for pointing it out. 🙌